crates/agglayer-config/src/rpc.rs

```
@@ -14,12 +18,24 @@ const DEFAULT_ADMIN_PORT: u16 = 9091;
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
 #[serde(rename_all = "kebab-case")]
 pub struct RpcConfig {
-    /// If the `PORT` environment variable is set, it will take precedence over
-    /// the configuration file.
-    #[serde(default = "default_port", deserialize_with = "deserialize_port")]
-    pub port: u16,
+    /// The default port for the local gRPC server.
+    /// Overridden by `AGGLAYER_GRPC_PORT` environment variable, defaults to 9089.
+    #[serde(
+        default = "default_grpc_port",
+        deserialize_with = "deserialize_grpc_port"
+    )]
+    pub grpc_port: u16,
 
-    /// The port for `admin` RPC
+    /// The default port for the local ReadRPC server.
+    /// Overridden by `AGGLAYER_READRPC_PORT` environment variable, defaults to 9090.
+    #[serde(
+        default = "default_readrpc_port",
```

Do we want to have an `alias = "port"` here for backward compatibility?

---

I'd rather not: better to fail loudly than silently accept deprecated options, especially today as we're still in a very centralized, easy-to-change setup.

Landing as-is, let me know if you think it's important and we can add the alias in a follow-up PR!